interface/src/app/mlmodel/mlModelComponents.js

```diff
@@ -30,4 +31,45 @@ angular.module('adage.mlmodel.components', [
   }]
 })
 
+.component('mlModelView', {
+  templateUrl: 'mlmodel/view.tpl.html',
+  controller: ['GlobalModelInfo', function(GlobalModelInfo) {
+    this.modelInfo = GlobalModelInfo;
+  }]
+})
+
+.component('mlModelValidator', {
```

Comment:
I'm not sure if this should be a component or just a service that is used by other components. What are the advantages of having it be a component that gets injected at the template level?